system-variables.md

```diff
@@ -775,6 +775,22 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 1 row in set (0.00 sec)
 ```
 
+### pd_enable_follower_handle_region <span class="version-mark">New in v7.6.0</span>
+
+> **Warning:**
+>
+> The [Active PD Follower](https://docs.pingcap.com/tidb/dev/tune-region-performance#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
```

Comment:
```suggestion

> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.

```

Reply:
@ CabinfeverB, please check if the variable is applicable to TiDB Cloud. If not, please add the above note.

Reply:
@qiancai pd_enable_follower_handle_region is also applicable to TiDB Cloud.